apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V53__Tech_debt_db_cleanup.sql

```diff
@@ -0,0 +1,13 @@
+-- both psql and hsql support non-primary key index renaming
+ALTER INDEX beneficiary_monthly_year_month_partd_contract_bene_id_idx RENAME TO beneficiary_monthly_partd_contract_year_month_bene_id_idx;
+
+-- to check if an index is not used in postgres:
+-- select * from pg_stat_all_indexes where schemaname = 'public' and relname = 'beneficiary_monthly'
```

Comment:
so this comes back with nothing in prod? did this get reset during the renaming migration though?

Reply:
so not sure what you are seeing; what happens if you exec the following in prod db?

`select * from pg_stat_user_indexes
where schemaname = 'public'
and relname = 'beneficiary_monthly'`

See the image I've added to this PR; it shows an index
`beneficiary_monthly_year_month_partd_contract_bene_id_idx`

Reply:
![Screen Shot 2022-02-07 at 3 37 40 PM](https://user-images.githubusercontent.com/28897779/152868539-cd1b501e-4b53-41f0-808b-0a50155db71b.png)

Reply:
thanks. I had not run it yet. this looks good to me